lib/events/EventManager.ts

```diff
@@ -232,8 +262,12 @@ export default class EventManager {
   private createVideoEvent(event: CalendarEvent, maybeUid?: string): Promise<EventResult> {
     const credential = this.getVideoCredential(event);
 
-    if (credential) {
+    const isDaily = event.location === "integrations:daily";
```

Comment:
I see we're using this string everywhere. Maybe we can extract this into a `CONSTANT` ?